plugins/generic/dublinCoreMeta/locale/fr_CA/locale.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/fr_CA/locale.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
```

Comment:
This copyright notice (and a few others below) still need updates:
- Remove the "Library'
- Update the end dates to 2018